### PR TITLE
[T4A2][T13-B2] Gao Yu

### PR DESCRIPTION
**Pull request by GaoYu-Karen**

### CODE REVIEW

**Comment by GaoYu-Karen** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -25,6 +29,8 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.nextSequenceNumber += 1;
+        System.out.println("Sequence number is " + this.sequenceNumber);
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Person.java:33]: good that you are using the generated sequence number in this method.
however, you are calling a raw print function here, recollect that there is an object specifically used to do user interaction, try using the sequence number in some class that uses textUi.

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -222,7 +222,7 @@ private int parseArgsAsDisplayedIndex(String args) throws ParseException, Number
      * @param args full command args string
      * @return the prepared command
      */
-    private Command prepareFind(String args) {
+    private static Command prepareFind(String args) {
```
Comment (okkhoy) [inline src/seedu/addressbook/parser/Parser.java:225]: you need to update parser test file as well since it uses `parseCommand` method

**Comment by okkhoy** [comment]
@GaoYu-Karen 
Some comments added. Please ack & close the PR after reading comments.

**Pull request closed without merging**